container/lxd/server.go

```diff
@@ -319,5 +319,5 @@ func (s *Server) SupportedArches() []string {
 // IsLXDNotFound checks if an error from the LXD API indicates that a requested
 // entity was not found.
 func IsLXDNotFound(err error) bool {
-	return err != nil && err.Error() == "not found"
+	return err != nil && (err.Error() == "not found" || err.Error() == "Not Found")
```

Comment:
I'm ok with this as a stop gap, but it sounds like Stephane already has typed errors that we could be using. Is there some sort of:
lxderr, ok := err.(lxd.Error)
that we should be using instead of string comparison here?
Or even
`strings.Lower(err.Error())` seems better than 2 different capitalizations.